bodhi/server/models.py

```diff
@@ -3340,12 +3341,10 @@ def comment(self, session, text, karma=0, author=None, karma_critpath=0,
             user = User(name=author)
             session.add(user)
 
-        comment = Comment(text=text, karma=karma, karma_critpath=karma_critpath, user=user)
+        comment = Comment(text=text, karma=karma, karma_critpath=karma_critpath,
```

Comment:
Why is the `update=self` needed now?

Reply:
Because after setting nullable=False on the update_id property it is now not possible to create a Comment object without specifying the associated update.